### ISSUE TITLE
[New Feature]: Support PackageManifest version 1.6.0 in package validation

### ISSUE THREAD

**Issue by williamoconnorme** [microsoft/winget-pkgs#121127]
### Description of the new feature/enhancement

WinGetSvc-Validation (Pull Request Validation) should support package manifest version 1.6.0

Currently the latest version supported is 1.5.0

This prevent packages from being submitted with the 1.6.0 schema

> One or more errors occurred. (Manifest: The manifest version is not supported. Supported versions: 1.0.0,1.1.0,1.2.0,1.4.0,1.5.0.) Manifest: The manifest version is not supported. Supported versions: 1.0.0,1.1.0,1.2.0,1.4.0,1.5.0.

https://github.com/microsoft/winget-pkgs/pull/120285/checks?check_run_id=17235916269

**Comment by BrandonWanHuanSheng** [comment]
@denelon

**Comment by mdanish-kh** [comment]
Can I ask what capability you're trying to use that's only available in the 1.6 schema? I believe the only new fields in 1.6 schema currently are `UpgradeBehavior: deny`, and `DownloadCommandProhibited: <boolean>`

**Comment by williamoconnorme** [comment]
With the release of CLI 1.6, I'm looking to use the `Dependencies` field which I seen was a feature of CLI 1.6

I can see now that it's available in schema 1.5.0 so I may not require this for my use case any more as I previously thought this feature was a part of the schema version matching CLI version



  I can see that it looks like this is available in 1.5.0 schema now.

**Comment by stephengillie** [comment]
Hi @williamoconnorme,

I believe there might not be a `1.6.0` manifest version, as there's no `1.2.0` manifest version. 

Here is a [1.5.0 Schema](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.5.0/installer.md) dependency example using our most common dependency:
```
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
- This can be used in the root of the Installers file, to apply to all installers. 
  - When doing this, it's commonly placed just above the `Installers:` field. 
  - The above section can be used as-is.
- Alternatively, this can be specified foreach installer, for example to have a 32-bit dependency for an `x86` package and the 64-bit complement for an `x64` package. 
  - When setting this way, it's common to add below the `InstallerSha256:` line.
  - Please remember to indent each line of the above section by 2 spaces (`  `) to match the installer section indent. As YAML is whitespace-delineated. 

Does this help?

**Comment by R-Adrian** [comment]
> Can I ask what capability you're trying to use that's only available in the 1.6 schema? I believe the only new fields in 1.6 schema currently are `UpgradeBehavior: deny`, and `DownloadCommandProhibited: <boolean>`


`UpgradeBehavior: deny` is needed for many of the apps that handle their own upgrades internally and that react badly / weirdly to being installed again instead of letting them handle the upgrades:

- Discord.Discord https://github.com/microsoft/winget-pkgs/issues/90642
- Mozilla.Firefox.DeveloperEdition https://github.com/microsoft/winget-pkgs/issues/79299  and https://bugzilla.mozilla.org/show_bug.cgi?id=1853673
- Chocolatey.Chocolatey  https://github.com/microsoft/winget-pkgs/pull/122574#issuecomment-1763158979
- TeamViewer.TeamViewer  - this one has an almost malware-ish behaviour imho - any upgrade (even internally in the app) will forcefully (re-)enable a system service with Automatic startup type even if the user had previously set that service to "Disabled" startup type, and that service runs elevated all the time as Local System account, and the upgrade process does this even if the user preference in-app is to not start it with Windows at all.

**Comment by denelon** [comment]
1.6 validation is in production.
